addons/web/static/src/js/views/control_panel/control_panel_model.js

```diff
@@ -497,12 +498,14 @@ var ControlPanelModel = mvc.Model.extend({
             const combinationId = [filterId, optionId];
```

Comment:
this is not an id

Reply:
It is called that way elsewhere and it is going to change in master